gitbootcamp.rst

```diff
@@ -259,8 +256,17 @@ Solution::
 
    git checkout some-branch
    git fetch upstream
-   git rebase upstream/master
-   git push --force origin some-branch
+   git merge upstream/master
+   git push origin some-branch
+
+You may see error messages like "CONFLICT" and "Automatic merge failed;" when
+you run ``git merge upstream/master``.
+
+When it happens, you need to resolve conflict.  See following articles for
+how to resolve conflicts.
```

Comment:
Could anyone check my English?
* happens vs happened
* "happens, " -- this comma is usual?
* "for how to..." -- is this usual wording?

Reply:
* You're using "happens" correctly.

* The common after "happens" is a typical usage.

* I'd say something more like "See these articles about resolving conflicts:".  The "for how to" is a little awkward.

Reply:
Thanks!